packages/php-domain/src/TextRoles/InterfaceTextRole.php

```diff
@@ -14,7 +14,13 @@
 
 final class InterfaceTextRole implements TextRole
 {
+    /**
+     * @see https://regex101.com/r/OyN05v/1
+     */
+    private const INTERFACE_NAME_REGEX = '/^([a-zA-Z0-9]+):(.*$)/';
```

Comment:
```suggestion
    private const INTERLINK_NAME_REGEX = '/^([a-zA-Z0-9]+):(.*$)/';
```

We are matching for interlinks here which would be devided by a colon from the interface name. We are not matching for interfaces. This can be used to link to an interface in another manuals i.e. 

```
:php:interface:`someManual:\MyVendor\MyProject\SomeInterface`